test/v2/removeLiquidity/composableStable.recovery.integration.test.ts

```diff
@@ -20,17 +19,18 @@ import {
     walletActions,
 } from 'viem';
 import { startFork, ANVIL_NETWORKS } from 'test/anvil/anvil-global-setup';
-import { InputAmount, PoolType } from '@/types';
+import { InputAmount } from '@/types';
 import {
     doRemoveLiquidity,
     assertRemoveLiquidityRecovery,
 } from 'test/lib/utils/removeLiquidityHelper';
 import { test } from 'vitest';
+import { POOLS, TOKENS } from 'test/lib/utils/addresses';
 
 const chainId = ChainId.MAINNET;
 const { rpcUrl } = await startFork(ANVIL_NETWORKS.MAINNET);
-const poolId =
-    '0x156c02f3f7fef64a3a9d80ccf7085f23cce91d76000000000000000000000570'; // swETH-bb-a-WETH-BPT
+const TOKENS_MAINNET = TOKENS[chainId];
+const POOLS_MAINNET = POOLS[chainId];
```

Comment:
Something that could make it cleaner and easier to replace the pool if needed would be to declare a testPool and reuse through the test:
```suggestion
const testPool = POOLS[chainId].swETH_bb_a_WETH_BPT;
```